util.js

```diff
@@ -106,7 +106,7 @@ exports.deprecate = function(fn, msg) {
 var debugs = {};
 var debugEnvRegex = /^$/;
 
-if (process.env.NODE_DEBUG) {
+if (typeof process !== 'undefined' && process.env && process.env.NODE_DEBUG) {
```

Comment:
```suggestion
if (typeof process !== 'undefined' && process.env.NODE_DEBUG) {
```
if process exists, it's node, and `process.env` should also exist

Reply:
Yeah that's true, I was being over defensive here